src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseConcreteTypeAnalyzer.Collector.cs

```diff
@@ -85,7 +85,70 @@ public static void ReturnInstance(Collector c, CancellationToken cancellationTok
             }
 
             /// <summary>
-            /// Identify fields/locals/params that are used as 'this' for a virtual dispatch.
+            /// Identify properties that are used as 'this' for a virtual dispatch.
+            /// </summary>
+            public void HandlePropertyReference(IPropertyReferenceOperation op)
+            {
+                if (op.Property.IsAbstract || op.Property.IsVirtual)
+                {
+                    if (op.Instance != null)
+                    {
+                        var targetMethod = op.Property.GetMethod ?? op.Property.SetMethod!;
+
+                        var instance = op.Instance;
+                        if (instance.Kind == OperationKind.ConditionalAccessInstance)
+                        {
+                            var parent = ((IConditionalAccessInstanceOperation)instance).GetConditionalAccess() ?? instance;
+                            if (parent != null)
+                            {
+                                instance = ((IConditionalAccessOperation)parent).Operation;
+                            }
+                        }
```

Comment:
Did we add a test validating conditional access?

Reply:
I added a test now.